File: CompSci 1/Unit 12, Selection/LetterGrade/LetterGrade.java
import java.util.Scanner;

public class LetterGrade {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a numerical grade --> ");
        double n = sc.nextDouble();
        System.out.print("\nLetter grade = ");

        if (n >= 97) {
            System.out.println("A+");
        }
        else if (n >= 94) {
            System.out.println("A");
        }
        else if (n >= 90) {
            System.out.println("A-");
        }
        else if (n >= 87) {
            System.out.println("B+");
        }
        else if (n >= 84) {
            System.out.println("B");
        }
        else if (n >= 80) {
            System.out.println("B-");
        }
        else if (n >= 77) {
            System.out.println("C+");
        }
        else if (n >= 74) {
            System.out.println("C");
        }
        else if (n >= 70) {
            System.out.println("C-");
        }
        else {
            System.out.println("F");
        }
    }
}

File: CompSci 1/Unit 12, Selection/LetterGrade/LetterGrade.kt
import java.util.Scanner

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)

    print("Enter a numerical grade --> ")
    val n = sc.nextDouble()
    print("\nLetter grade = ")

    if (n >= 97) {
        println("A+")
    } else if (n >= 94) {
        println("A")
    } else if (n >= 90) {
        println("A-")
    } else if (n >= 87) {
        println("B+")
    } else if (n >= 84) {
        println("B")
    } else if (n >= 80) {
        println("B-")
    } else if (n >= 77) {
        println("C+")
    } else if (n >= 74) {
        println("C")
    } else if (n >= 70) {
        println("C-")
    } else {
        println("F")
    }
}
